### PR TITLE
feat(Quantic): fix triggering query suggestions for any key press

### PR DESCRIPTION
**Pull request by jelmedini**
Fixing unnecessary calls when typing, calling the api only when the value is changed

### CODE REVIEW

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js [hunk 1]
```diff
@@ -210,6 +210,13 @@ export default class QuanticSearchBox extends LightningElement {
     this.input.blur();
   }
 
+  handleKeyValues() {
+    if (this.searchBox.state.value !== this.input.value) {
+      this.suggestionList?.resetSelection();
+      this.searchBox.updateText(this.input.value);
+    }
+  }
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:218]: Should we actually have a debounce as well?
Like a small delay before calling the suggestions?

**Review by erocheleau — APPROVED** [review]
Just added a question, but your changes are good

**Review by mmitiche — APPROVED** [review]
LGTM👌

File: packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js (hunk 1 above)
Comment (jelmedini) [reply packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js:218]: I don't know if it s relevant here regarding how the component should work, but if so we need to do this improvement in several locations on this file

**Pull request merged**